src/main/java/ai/elimu/web/content/word/WordEditController.java

```diff
@@ -88,13 +88,13 @@ public String handleRequest(
         Word word = wordDao.read(id);
         
         if (word.getLetterSounds().isEmpty()) {
-            autoSelectLetterSoundCorrespondences(word);
+            autoSelectLetterSounds(word);
             // TODO: display information message to the Contributor that the letter-sound correspondences were auto-selected, and that they should be verified
         }
                 
         model.addAttribute("word", word);
         model.addAttribute("timeStart", System.currentTimeMillis());
-        model.addAttribute("letterSoundCorrespondences", letterSoundDao.readAllOrderedByUsage()); // TODO: sort by letter(s) text
+        model.addAttribute("letterSounds", letterSoundDao.readAllOrderedByUsage()); // TODO: sort by letter(s) text
```

Comment:
Word editing at http://localhost:8080/webapp/content/word/edit/<ID> stopped working because of this change. So please also update the controller's frontend file at `src/main/webapp/WEB-INF/jsp/content/word/edit.jsp`.

<kbd>![](https://github.com/user-attachments/assets/e9989368-d1a8-4f64-b6a3-864dceed5873)</kbd>